rider/src/main/kotlin/com/jetbrains/rider/plugins/unity/run/UnityDebuggerOutputListener.kt

```diff
@@ -2,30 +2,55 @@ package com.jetbrains.rider.plugins.unity.run
 import com.intellij.notification.NotificationType
 import com.intellij.openapi.project.Project
+import com.intellij.util.text.VersionComparatorUtil
 import com.intellij.xdebugger.XDebuggerManager
 import com.intellij.xdebugger.impl.XDebuggerManagerImpl
+import com.jetbrains.rd.util.reactive.fire
 import com.jetbrains.rider.debugger.DotNetDebugProcess
 import com.jetbrains.rider.model.debuggerWorker.OutputMessageWithSubject
 import com.jetbrains.rider.model.debuggerWorker.OutputSubject
 import com.jetbrains.rider.model.debuggerWorker.OutputType
+import com.jetbrains.rider.model.rdUnityModel
+import com.jetbrains.rider.plugins.unity.util.UnityInstallationFinder
+import com.jetbrains.rider.projectView.solution
 import com.jetbrains.rider.run.IDebuggerOutputListener
+import javax.swing.event.HyperlinkEvent
 
 class UnityDebuggerOutputListener(val project: Project, private val host: String, private val targetName: String, private val isEditor: Boolean)
     : IDebuggerOutputListener {
 
     override fun onOutputMessageAvailable(message: OutputMessageWithSubject) {
         if (message.subject == OutputSubject.ConnectionError) {
             var text = "Unable to connect to $targetName"
-            text += if (isEditor) {
-                "\nPlease ensure 'Editor Attaching' is enabled in Unity's External Tools settings page.\n"
-            }
-            else {
-                "\nPlease ensure that the player has 'Script Debugging' enabled and that the host '$host' is reachable.\n"
+
+            val unityVersion: String? = UnityInstallationFinder.getInstance(project).getApplicationVersion(2)
+            if (unityVersion != null && VersionComparatorUtil.compare(unityVersion, "2018.2") >= 0) {
+                if (isEditor) {
+                    text += "\nPlease follow <a href=\"https://docs.unity3d.com/$unityVersion/Documentation/Manual/ManagedCodeDebugging.html\">Debugging in the Editor</a> documentation.\n"
+                } else {
+                    text += "\nPlease follow <a href=\"https://docs.unity3d.com/$unityVersion/Documentation/Manual/ManagedCodeDebugging.html\">Debugging in the Player</a> documentation.\n"
+                }
+            } else
+                if (isEditor) {
+                    text += "\nPlease ensure 'Editor Attaching' is enabled in Unity's External Tools settings page.\n"
+                } else {
+                    text += "\nPlease ensure that the player has 'Script Debugging' enabled and that the host '$host' is reachable.\n"
+                }
+
+            val debugNotification = XDebuggerManagerImpl.NOTIFICATION_GROUP.createNotification(text, NotificationType.ERROR)
+
+            debugNotification.setListener { notification, hyperlinkEvent ->
+                if (hyperlinkEvent.eventType != HyperlinkEvent.EventType.ACTIVATED)
+                    return@setListener
+
+                BrowserUtil.browse("https://docs.unity3d.com/$unityVersion/Documentation/Manual/ManagedCodeDebugging.html")
```

Comment:
Can this by `hyperlinkEvent.getURL()`? Would be nice to just use the URL you've already specified rather than have it in 3 different places.